LanesReminder/VideoTableViewCell.h

```diff
@@ -0,0 +1,19 @@
+//
+//  VideoTableViewCell.h
+//  LanesReminder
+//
+//  Created by Sanjeev_Bharti on 5/4/16.
+//  Copyright © 2016 Sanjeev_Bharti. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
```

Comment:
Minor, remove objc warning by giving two lines spaces before declaration of a class